rust-src/concordium_base/src/hashes.rs

```diff
@@ -143,6 +143,10 @@ impl<Purpose> From<[u8; 32]> for HashBytes<Purpose> {
     fn from(array: [u8; 32]) -> Self { Self::new(array) }
 }
 
+impl<Purpose> From<HashBytes<Purpose>> for [u8; 32] {
+    fn from(value: HashBytes<Purpose>) -> Self { value.bytes }
+}
+
```

Comment:
I needed this as well and made this this PR https://github.com/Concordium/concordium-base/pull/325/files some days ago.

In light of that I think we should remove this.